src/caselawclient/Client.py

```diff
@@ -110,6 +121,25 @@ def get_single_string_from_marklogic_response(
     return parts[0]
 
 
+def get_single_bytestring_from_marklogic_response(
```

Comment:
do we need all of this? could we not just encode the xml string into bytes if lxml is complaining about bytes?

Reply:
aha this came from the chat on slack earlier - @jacksonj04 and @dragon-dxw felt it was best if lxml expects bytes to just pass the raw response through directly

Reply:
Ah right, i just wonder if doing a .encode() is sufficient - stops us from needing to add and test new functions that arent used otherwise!